motor_control/throttle/base_throttle.ino

```diff
@@ -3,7 +3,6 @@
 Throttle::Throttle(const int &pin, bool digital)
 {
     m_sensor_pin = pin;
-    pinMode(m_sensor_pin, INPUT);
```

Comment:
Ahh this is really unfortunate, but since you added the `pinMode` line before the `m_sensor_pin = pin;` line, we're not really setting the mode on the correct pin. You need to swap the order of those two lines if that makes sense.